Tiltfile

```diff
@@ -382,21 +382,24 @@ def get_addons(flavor_name):
     if "aks" in flavor_name:
         return ""
 
-    # install calico
-    if "ipv6" in flavor_name:
-        calico_values = "./templates/addons/calico-ipv6/values.yaml"
-    elif "dual-stack" in flavor_name:
-        calico_values = "./templates/addons/calico-dual-stack/values.yaml"
-    else:
-        calico_values = "./templates/addons/calico/values.yaml"
-
-    addon_cmd = "; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://docs.tigera.io/calico/charts --version ${CALICO_VERSION} calico tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace"
-
+    addon_cmd = ""
     if "intree-cloud-provider" not in flavor_name:
-        addon_cmd += "; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName=${CLUSTER_NAME}"
+        addon_cmd += "; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName=${CLUSTER_NAME} --set cloudControllerManager.clusterCIDR='192.168.0.0/16'"
```

Comment:
Is there a way we could avoid hardcoding `cloudControllerManager.clusterCIDR` to `192.168.0.0/16` by setting it dynamically based on the actual value of the `Cluster`?

Reply:
```
addon_cmd += "; export CIDRS=$(" + kubectl_cmd + " get cluster ${CLUSTER_NAME} -o jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[*]}')"
addon_cmd += "; export CIDR_LIST=$(bash -c 'echo $CIDRS' | tr ' ' ',')"
```

Should now get `CIDR_LIST` and add to the helm command dynamically.